index.html

```diff
@@ -38,17 +41,180 @@ <h1 >
+                </p>
+                <ul class="technology-tags">
+                    <li>HTML</li>
+                    <li>CSS</li>
+                    <li>Javascript</li>
+                </ul>
+                <button class="See-project-button" type="button">See Project</button>
+            </div>
+            <div class="card-element">
+                <img class="preview-picture" src="./img/Snapshoot Portfolio2.svg" alt="Preview Project">
+                <h2 class="project-name">Multi-Post
+                    Stories</h2>
+                <ul class="projext-frame">
+                    <li>CANOPY</li>
+                    <li>Back End Dev</li>
+                    <li>2015</li>
+                </ul>
+                <p class="card-main-text">
+                    A daily selection of privately personalized reads; no accounts
+                    or sign-ups required.
+                </p>
+                <ul class="technology-tags">
+                    <li>HTML</li>
+                    <li>CSS</li>
+                    <li>Javascript</li>
+                </ul>
+                <button class="See-project-button" type="button">See Project</button>
+            </div>
+            <div class="card-element">
+                <img class="preview-picture" src="./img/Snapshoot Portfolio3.svg" alt="project preview">
+                <h2 class="project-name">Tonic</h2>
+                <ul class="projext-frame">
+                    <li>CANOPY</li>
+                    <li>Back End Dev</li>
+                    <li>2015</li>
+                </ul>
+                <p class="card-main-text">
+                    A daily selection of privately personalized reads; no accounts
+                    or sign-ups required.
+                </p>
+                <ul class="technology-tags">
+                    <li>HTML</li>
+                    <li>CSS</li>
+                    <li>Javascript</li>
+                </ul>
+                <button class="See-project-button" type="button">See Project</button>
+            </div>
+            <div class="card-element">
+                <img class="preview-picture" src="./img/Snapshoot Portfolio4.svg" alt="project preview">
+                <h2 class="project-name">Multi-Post Stories</h2>
+                <ul class="projext-frame">
+                    <li>CANOPY</li>
+                    <li>Back End Dev</li>
+                    <li>2015</li>
+                </ul>
+                <p class="card-main-text">
+                    A daily selection of privately personalized reads; no accounts
+                    or sign-ups required.
+                </p>
+                <ul class="technology-tags">
+                    <li>HTML</li>
+                    <li>CSS</li>
+                    <li>Javascript</li>
+                </ul>
+                <button class="See-project-button" type="button">See Project</button>
+            </div>
+
+
+
+
+
+        </div>
+    </div>
```

Comment:
[OPTIONAL] Kindly I suggest you use the section tag for the main container of works-wrap, to follow HTML semantic and best practices. 😄